javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+
+var fullName = 'Sarah Ryu Ocampo';
+var isCool = true;
+var totalPets = 1;
+var unknown;
+var never =
```

Comment:
If you are assigning an empty value to a variable you are declaring, the syntax should be like this: `var never;`